nautobot_ssot/migrations/0006_ssotservicenowconfig.py

```diff
@@ -41,6 +41,8 @@ def _move_data(apps, schema_editor):
 
 
 class Migration(migrations.Migration):
+    """Migration class."""
```

Comment:
These should not be needed, we should update the configuration for pydocstyle's ruff equivalant to not need doc strings.